src/components/List/List.md

```diff
@@ -0,0 +1,34 @@
+# List
+Presents a number of related items, either as a vertical list or in a grid.
```

Comment:
Ah, I get that you're commenting on the *list* and not the nuances of the individual list items.